### ISSUE TITLE
Output csv file

### ISSUE THREAD

**Issue by ZorigTsogt** [TadasBaltrusaitis/OpenFace#920]
Hi, thank you so much for the work you have produced. This isn't a bug but I am having an issue with finding out how to edit the data the csv file outputs after processing an image. I am using the Windows GUI to output my data and I want to be able to see the file name in the csv data after using processing a directory of images as a sequence. Right now it only outputs frame number which makes it confusing to see which data is for which image. Any help would be greatly appreciated.

**Comment by YunShen66666** [comment]
> Hi, thank you so much for the work you have produced. This isn't a bug but I am having an issue with finding out how to edit the data the csv file outputs after processing an image. I am using the Windows GUI to output my data and I want to be able to see the file name in the csv data after using processing a directory of images as a sequence. Right now it only outputs frame number which makes it confusing to see which data is for which image. Any help would be greatly appreciated.

Hi, I meet the same question. The frame information are not enough. And I can't find which frame Correspond to which image. Did you have some solution ?

**Comment by YunShen66666** [comment]
> Hi, thank you so much for the work you have produced. This isn't a bug but I am having an issue with finding out how to edit the data the csv file outputs after processing an image. I am using the Windows GUI to output my data and I want to be able to see the file name in the csv data after using processing a directory of images as a sequence. Right now it only outputs frame number which makes it confusing to see which data is for which image. Any help would be greatly appreciated.

Hi. I find the solution method.  *Only* `FaceLandmarkImg` is used to extract single image.And it will output the `csv` file with the image name. In other words, other executables are used for vidio so that the only output frames. 

A example: If you have some images named 111.jpg and 222.jpg in the image directionary, using `FaceLandmarkImg -fdir image` will output 111.csv and 222.csv ,which is corresponding to 111.jpg and 222.jpg.

**Comment by onurbarut** [comment]
Hi, is there any documentation regarding the units of columns in the output csv file?

**Comment by healyr4** [comment]
> Hi, is there any documentation regarding the units of columns in the output csv file?
Yes, they are in the Wiki [here](https://github.com/TadasBaltrusaitis/OpenFace/wiki/Output-Format)
